rest_framework_gis/filters.py

```diff
@@ -40,34 +40,45 @@
 class InBBoxFilter(BaseFilterBackend):
     bbox_param = 'in_bbox'  # The URL query parameter which contains the bbox.
 
-    def get_filter_bbox(self, request):
-        bbox_string = request.query_params.get(self.bbox_param, None)
+    def get_filter_bbox(self, request, query_param):
+        bbox_string = request.query_params.get(query_param, None)
         if not bbox_string:
             return None
 
         try:
             p1x, p1y, p2x, p2y = (float(n) for n in bbox_string.split(','))
         except ValueError:
-            raise ParseError('Invalid bbox string supplied for parameter {0}'.format(self.bbox_param))
+            raise ParseError('Invalid bbox string supplied for parameter {0}'.format(query_param))
 
         x = Polygon.from_bbox((p1x, p1y, p2x, p2y))
         return x
 
     def filter_queryset(self, request, queryset, view):
-        filter_field = getattr(view, 'bbox_filter_field', None)
-        include_overlapping = getattr(view, 'bbox_filter_include_overlapping', False)
-        if include_overlapping:
-            geoDjango_filter = 'bboverlaps'
+        query_filter = {}
+        self.bbox_params = getattr(view,'bbox_params', {self.bbox_param: {'filter_field': getattr(view,'bbox_filter_field', None), 'include_overlapping': getattr(view,'bbox_filter_include_overlapping', False)}})
+        for query_param, filter_field_attributes in self.bbox_params.iteritems():
```

Comment:
this will need to be something like `six.iteritems(self.bbox_params)` for 2 and 3 compatibility (if six isn't available I think django has a copied version of it somewhere)